### PR TITLE
fix: invalid id for cts, mts, cjs, mjs files

### PR DESCRIPTION
**Pull request by jacoobes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "@sern/handler",
     "packageManager": "yarn@3.5.0",
-    "version": "3.0.1",
+    "version": "3.0.2",
```
Comment (EvolutionX-10) [inline package.json:4]: are you manually bumping verions now?

Comment (jacoobes) [reply package.json:4]: Oh oops, ur right I should revert

Tomorrows problem 💤

**Review by EvolutionX-10 — APPROVED** [review]
lgtm

**Comment by jacoobes** [comment]
Will revert the manual version bump and then merge this. Workflows should take care of publishing and making release notes

**Pull request merged**